recipes/http-classes-with-new/README.md

```diff
@@ -0,0 +1,35 @@
+# `http.request` DEP0195
+
+This recipe provides a guide for migrating from the deprecated `http.request` and its synchronous and promise-based counterparts to the new `http.request` method in Node.js.
+
+See [DEP0195](https://nodejs.org/api/deprecations.html#DEP0195).
+
+## Examples
+
+**Before:**
+
+```js
+const http = require("node:http");
+const message = http.OutgoingMessage();
+const response = http.ServerResponse(socket);
+```
+
+```js
+import { IncomingMessage, ClientRequest } from "node:http";
+const incoming = IncomingMessage(socket);
+const request = ClientRequest(options);
+```
```

Comment:
Nit: This migration does not mutate requires/imports, so this is only meta-info, so I would write it as a comment and combine these two code blocks since require vs import is irrelevant.


```suggestion
```js
// import { IncomingMessage, ClientRequest } from "node:http";
// const http = require("node:http");

const message = http.OutgoingMessage();
const response = http.ServerResponse(socket);

const incoming = IncomingMessage(socket);
const request = ClientRequest(options);
```
```